phoenix-core/src/main/java/org/apache/phoenix/coprocessor/generated/ServerCachingProtos.java

```diff
@@ -2192,6 +2192,21 @@ public Builder setColumnNameBytes(
      */
     com.google.protobuf.ByteString
         getParentTableTypeBytes();
+
+    // optional string logicalIndexName = 25;
+    /**
+     * <code>optional string logicalIndexName = 25;</code>
+     */
+    boolean hasLogicalIndexName();
+    /**
+     * <code>optional string logicalIndexName = 25;</code>
+     */
+    java.lang.String getLogicalIndexName();
```

Comment:
Would this be populated for both view indexes and regular indexes and local indexes?

Reply:
@gjacoby126 thanks for addressing my comments. There is just this question remaining.

Reply:
Correct, should be populated for all indexes.